portal/gil/templates/gil/base.html

```diff
@@ -278,6 +278,40 @@ <h4 class="box-modal__title">{{_("Session Timed Out")}}</h4><p class="box-modal_
       </div>
     </div>
 {% endblock %}
+{# One-time system messages called Flash messages #}
+{% block flash_messages %}
+  {%- with messages = get_flashed_messages(with_categories=true) -%}
+      {% if messages %}
+          {# Hack to prevent modal if it's only successful login #}
+          {% set showMessages = true %}
+          {% if messages|first|last == "You have signed in successfully." %}
```

Comment:
Will this work if viewing non-english?

Reply:
@pbugni paul, i think it will work, but not sure.. messages variable were assigned the text (e.g. "You have signed in successfully.") before it is translated later.  @ivan-c Ivan, am I correct here?

Reply:
It should work as long as `messages` are translated beforehand, but I can't find where `flash(_("You have signed in successfully."))` is being called. Where is that string coming from @achen2401 ?

[See](http://flask.pocoo.org/docs/0.12/patterns/flashing/)

Reply:
@ivan, I think it is coming from part of flask user library - see /truenth/env/src/flask-user/flask_user/views.py, _do_login_user function

Reply:
Well -  let's not hold it up for now, sounds like you believe this will work in either case.  We should simply test in another language.

@ivan-c that line comes from https://github.com/lingthio/Flask-User/blob/v0.6/flask_user/views.py#L682  (do we need to consider external libraries in building .pot files?)